nvtabular/loader/tensorflow.py

```diff
@@ -215,7 +215,7 @@ def __init__(
         # sort the ccolumns to avoid getting incorrect output
         # (https://github.com/NVIDIA/NVTabular/issues/412)
         cat_names = _get_embedding_order(cat_names)
-        cont_names = _get_embedding_order(cont_names)
```

Comment:
I don't think we can get rid of the sorting here ... if we add a similar test for the ordering for the continuous columns, this change will cause incorrect results.


Can we revert this change and add this patch to the unittest?
```
diff --git a/tests/unit/test_tf_dataloader.py b/tests/unit/test_tf_dataloader.py
index 90dca67..cfccc27 100644
--- a/tests/unit/test_tf_dataloader.py
+++ b/tests/unit/test_tf_dataloader.py
@@ -32,12 +32,13 @@ tf_dataloader = pytest.importorskip("nvtabular.loader.tensorflow")
 
 def test_tf_catname_ordering(tmpdir):
     df = cudf.DataFrame(
-        {"cat1": [1] * 100, "cat2": [2] * 100, "cat3": [3] * 100, "label": [0] * 100}
+        {"cat1": [1] * 100, "cat2": [2] * 100, "cat3": [3] * 100, "label": [0] * 100,
+         "cont3": [3.0] * 100, "cont2": [2.0] * 100, "cont1": [1.0] * 100 }
     )
     path = os.path.join(tmpdir, "dataset.parquet")
     df.to_parquet(path)
     cat_names = ["cat3", "cat2", "cat1"]
-    cont_names = []
+    cont_names = ["cont3", "cont2", "cont1"]
     label_name = ["label"]
 
     data_itr = tf_dataloader.KerasSequenceLoader(
@@ -54,6 +55,10 @@ def test_tf_catname_ordering(tmpdir):
         assert list(X["cat2"].numpy()) == [2] * 10
         assert list(X["cat3"].numpy()) == [3] * 10
 
+        assert list(X["cont1"].numpy()) == [1.0] * 10
+        assert list(X["cont2"].numpy()) == [2.0] * 10
+        assert list(X["cont3"].numpy()) == [3.0] * 10
+
 
 # TODO: include use_columns option
 # TODO: include parts_per_chunk test
```